### DISCUSSION TITLE
Is it possible to publish a manuscript project to gh-pages?

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#8616] [Q&A] [answered]
### Description

When I try to publish to Github, it complaints it is not a book nor website project.

```
❯ quarto publish gh-pages
ERROR: The specified path (/Users/fredguth/Library/CloudStorage/OneDrive-Personal/Projects/sus/1_Projects/Produto-1) is not a website or book project so cannot be published.
````
----
Using Quarto from repo (commit bf59e13de8cf32707354b63050aa251536bbb8d8)
❯ quarto check
Check file:///Users/fredguth/Resources/quarto-cli/src/quarto.ts
Quarto 99.9.9
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.11: OK
      Dart Sass version 1.69.5: OK
      Deno version 1.37.2: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 99.9.9
      Path: /Users/fredguth/Resources/quarto-cli/package/dist/bin

[✓] Checking tools....................OK
      TinyTeX: v2023.01
      Chromium: 869685

[✓] Checking LaTeX....................OK
      Using: TinyTex
      Path: /Users/fredguth/Library/TinyTeX/bin/universal-darwin
      Version: 2022

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.11.7
      Path: /opt/homebrew/opt/python@3.11/bin/python3.11
      Jupyter: 5.3.1
      Kernels: python3

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........OK
      Version: 4.3.2
      Path: /opt/homebrew/Cellar/r/4.3.2/lib/R
      LibPaths:
        - /opt/homebrew/lib/R/4.3/site-library
        - /opt/homebrew/Cellar/r/4.3.2/lib/R/library
      knitr: (None)
      rmarkdown: (None)

      The knitr package is not available in this R installation.
      Install with install.packages("knitr")
      The rmarkdown package is not available in this R installation.
      Install with install.packages("rmarkdown")

**Comment by cderv** [comment, marked as answer]
Our error message is wrong here as Manuscripts are supported. (we'll improve it). 

Manuscripts are checked specifically in our code 
https://github.com/quarto-dev/quarto-cli/blob/1dfc6db4c3462dfe164e1c8364be2d3315e6d5f8/src/command/publish/cmd.ts#L323-L325

I also tested it with a new manuscript project `quarto create project` and it works ok. Demo repo I just published https://github.com/cderv/test-manuscript

so what is your `_quarto.yml` content ? From where are you calling `quarto publish` command ?

**Reply by fredguth** [reply]
I am not sure what happened, now it works. It is possible that I was trying to publish from the `gh-pages` branch.  But it is indeed working. 

(using Quarto 99.9)
